scripts/update-tezos.sh

```diff
@@ -49,6 +49,10 @@ if [[ "$latest_upstream_tag" != "$our_tezos_tag" ]]; then
     git commit -a -m "[Chore] Reset release number for $latest_upstream_tag" --gpg-sign="tezos-packaging@serokell.io" || \
       echo "release number wasn't updated"
 
+    sed -i 's/letter_version *= *"[a-z]"/letter_version = ""/' ./docker/package/model.py
+    git commit -a -m "[Chore] Reset letter_version for $latest_upstream_tag" --gpg-sign="tezos-packaging@serokell.io" || \
```

Comment:
I'd just add a similar comment as the one on the release number above:
```suggestion
    # Commit may fail when the letter version wasn't updated since the last release
    git commit -a -m "[Chore] Reset letter_version for $latest_upstream_tag" --gpg-sign="tezos-packaging@serokell.io" || \
```